source/deployment-considerations.txt

```diff
@@ -0,0 +1,20 @@
+.. _rm-deployment-considerations:
+
+=========================
+Deployment Considerations
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The pages in this section outline considerations that should be taken 
+into account as you prepare to deploy Relational Migrator. 
```

Comment:
Can we shorten to:

The pages in this section help you prepare to deploy Relational Migrator.

?

Reply:
Updated.